src/main/views/et-daily-list.njk

```diff
@@ -0,0 +1,146 @@
+            <div class="parent-box overflow-table">
+              <table class="govuk-table">
+                <thead class="govuk-table__head">
+                <tr class="govuk-table__row">
+                  {% for header in tableHeaders %}
+                    <th scope="col" class="govuk-table__header">{{ header }}</th>
+                  {% endfor %}
+                </tr>
+                </thead>
+                <tbody class="govuk-table__body">
+                {% for session in courtRoom['session'] %}
+                  {% for sitting in session['sittings'] %}
+                    {% set durationAsHours = 0 %}
+                    {% if sitting['durationAsHours'] > 1 %}
+                      {% set durationAsHours = sitting['durationAsHours'] + ' ' + hearingDurationHours %}
+                      {% elif sitting['durationAsHours'] === 1 %}
+                      {% set durationAsHours = sitting['durationAsHours'] + ' ' + hearingDurationHour %}
+                    {% endif %}
+                    {% set durationAsMinutes = 0 %}
+                    {% if sitting['durationAsMinutes'] > 1 %}
+                      {% set durationAsMinutes = sitting['durationAsMinutes'] + ' ' + hearingDurationMinutes %}
+                      {% elif sitting['durationAsMinutes'] === 1 %}
+                      {% set durationAsMinutes = sitting['durationAsMinutes'] + ' ' + hearingDurationMinute %}
+                    {% endif %}
+                    {% set duration = '' %}
+                    {% if durationAsHours === 0 and durationAsMinutes === 0 %}
+                      {% set duration = '' %}
+                      {% elif durationAsHours !== 0 and durationAsMinutes !== 0 %}
+                      {% set duration = durationAsHours + ' ' + durationAsMinutes %}
+                      {% elif durationAsHours === 0 %}
+                      {% set duration = durationAsMinutes %}
+                      {% elif durationAsMinutes === 0 %}
+                      {% set duration = durationAsHours %}
+                    {% endif %}
+                    {% for hearing in sitting['hearing'] %}
+                      {% for case in hearing['case'] %}
+                        {% if courtRoom['totalHearing'] === hearingCount %}
+                          {% set noBorder = 'no-border-bottom' %}
+                        {% endif %}
+                        {% set caseIndicator = case['caseSequenceIndicator'] %}
+                        <tr class="govuk-table__row">
+                          <td class="govuk-table__cell {{ noBorder }}">{{ sitting['sittingStartFormatted'] }}</td>
+                          <td class="govuk-table__cell {{ noBorder }}">{{ duration }} {{ caseIndicator }}</td>
+                          <td class="govuk-table__cell {{ noBorder }}">{{ case['caseNumber'] }}</td>
+                          <td
+                            class="govuk-table__cell {{ noBorder }}">{{ hearing['claimant'] }}</td>
```

Comment:
Can place the 'td' tag on the same line here. Same for the next one

Reply:
will do.